book/website/communication/personal/personal-cv.md

```diff
@@ -0,0 +1,146 @@
+(cm-personal-narrative-cv)=
+# Narrative/Evidenced based CVs
+
+
+The 'narrative' or 'evidenced based' CV format provides a structured written description of a person’s contributions and achievements to research. 
+Compared to traditional academic CVs, narrative CVs reflect a broad range of relevant skills and experiences. 
+As there is no ideal type of researcher, evidence-based CVs allow you to highlight your own academic profile without being limited by a small range of questions and criteria.  
+
+(cm-personal-narrative-cv-motivation)=
+## Motivation
+
+Traditional academic CVs have focused primarily on the quantity rather than the quality of research contributions. 
+By focusing on a narrow range of metrics based on publications and obtainal of grants, traditional CVs do not capture the full range of research acitivities. 
+Traditional CVs also don't allow an explanation of how researchers have overcome barriers and how they embraced opportunities. 
+Instead, narrative CVs place more focus on how the activities you undertook strenghtened your skills or were important for your career stage, and the rationale for undertaking those activities.
+
+The narrative CV aims to reward a broader set of research-related activities beyond peer-reviewed publications, valuing diverse contributions and career paths in research. 
+This will lead to more inclusion and diversity in research, as well as encouraging a more responsible manner of using metrics. 
```

Comment:
This feels like a repetition of the above paragraph, and is duplicated from another page.